scss/_base_typography.scss

```diff
@@ -216,57 +187,193 @@
     @include vf-heading-6;
   }
 
-  li {
-    margin-bottom: 0;
-    margin-top: 0;
-    padding-bottom: 0;
-  }
-
-  li > ul,
-  li > ol {
-    padding-top: 0;
-  }
-
-  li > ul > li:last-of-type,
-  li > ol > li:last-of-type {
-    padding-bottom: 0;
-  }
-
   // blockquotes
   blockquote {
     margin-bottom: 0;
     margin-left: 0;
+    margin-top: 0;
+    overflow: auto; // include margins of children into it's own height
     padding-left: $sp-large;
 
     > p {
-      font-size: $sp-medium;
       font-style: italic;
-      margin-top: $sp-small;
     }
 
     > cite {
-      font-size: $sp-medium;
       font-style: normal;
-      margin-top: $sp-small;
     }
   }
 
   // misc
   strong {
-    font-weight: 400;
+    @extend %bold;
   }
 
   small {
-    font-size: .875rem;
+    @extend %small-text;
   }
 
   sub,
   sup {
-    font-size: .75em;
     line-height: 0;
     position: relative;
     vertical-align: baseline;
   }
+
+  .text {
```

Comment:
I thought it would be useful to add this to generic elements to bring copy inside a li or div into alignment with the baseline grid. like <div class="text"></div>. Otherwise, it would require wrapping the text inside the div in a p. Removing it now.